File: Programacion/Tema3.Java/Ejercicios_Basicos_Bucles/Ejercicio_1.java
public class Ejercicio_1 {
    public static void main(String[] args) {

        int i = 0;
        System.out.println("While");
        while (i < 10) {
            i++;
            System.out.println(i);
        }

        i = 0;

        System.out.println("Do While");
        do {
            i++;
            System.out.println(i);
        } while (i < 10);

        System.out.println("For");
        for (i = 1; i <= 10; i++) {
            System.out.println(i);
        }
    }
}

File: Unity/Scripts_Unity/Scripts/Ejercicio_1.cs
using UnityEngine;

public class Ejercicio_1 : MonoBehaviour
{
    private float speed = 5;

    // Update is called once per frame
    void Update()
    {
        float horizontalAxis = Input.GetAxis("Horizontal");
        float verticalAxis = Input.GetAxis("Vertical");

        Vector3 direction = horizontalAxis * Vector3.right + verticalAxis * Vector3.up;
        direction.Normalize();
        if (direction.x < 0)
        {
            direction *= 2;
        }
        Vector3 velocity = direction * speed;
        Vector3 displacement = velocity * Time.deltaTime;

        transform.position += displacement;
    }
}